cluster/app/visits.lua

```diff
@@ -1,7 +1,29 @@
 local crud = require("crud")
+local utils = require('app.utils')
 
 local function find_visits_by_pet_id(pet_id)
-    return crud.select("visits", {{"=", "pet_id", pet_id}}, { batch_size = 1000, prefer_replica = true })
+    -- Example of pagination implementation
+    local visits, err= crud.select("visits", {{"=", "pet_id", pet_id}}, { first = 1000 })
+    if err then
+        return nil, err
+    end
+
+    local visits_butch
+    local need_more_requests = #visits.rows >= 1000
+    while need_more_requests do
+        visits_butch, err = crud.select("visits", {{"=", "pet_id", pet_id}}, { after = visits.rows[#visits.rows], first = 1000 })
+        if err then
+            return nil, err
+        end
+
+        if #visits_butch.rows == 0 then
+            break
+        end
+
+        visits.rows = utils.merge_tables(visits.rows, visits_butch.rows)
+    end
+
+    return visits
```

Comment:
I don't get it.

I want you to add paging as example. Our application doesn't need this because we don't have infinity scrolling or page numbers in front-end part. Just add
```lua
local function find_visits_by_pet_id(pet_id, batch_size, after)
    batch_size = batch_size or 1000
    -- Example of pagination implementation
    return crud.select("visits", {{"=", "pet_id", pet_id}}, { first = batch, after=after })
end
```

Java code will still only pass pet_id, and batch_size and after wil be nil